src/modelfree/configs/multi/score.py

```diff
@@ -254,7 +254,7 @@ def debug_one_each_type(score):
         spec = {
             'config': {
                 PATHS_AND_TYPES: tune.grid_search(
-                    _env_agents(agents={env: (range(1), range(1)) for env in BANSAL_GOOD_ENVS}) +
+                    _env_agents(agents={env: ([1], [1]) for env in BANSAL_GOOD_ENVS}) +
```

Comment:
Is this making a functionality change, since now instead of [0, 1] we're just passing in [1]? What's the effect of/motivation for this?

Reply:
`list(range(1)) == [0]`, we need it to be [1]. I could also have done `range(1, 2)` to have the same effect but that's messy.